src/ts/src/common/api/index.tsx

```diff
@@ -84,7 +84,7 @@ interface TreeResponse extends APIResponse {
 const TREE_MAP = new Map<string, keyof Tree>([
     ["phylo_tree_id", "id"],
     ["pathogen_genome_count", "pathogenGenomeCount"],
-    ["completed_date", "dateCompleted"],
+    ["completed_date", "creationDate"],
```

Comment:
should `completed_date` be changed to `creation_date`?

Reply:
that's what the design referred to it as so :woman_shrugging: